refnx/reduce/platypusnexus.py

```diff
@@ -498,6 +498,142 @@ def _chopper_values(self, h5data):
         return master, slave, speeds[0] / 60.0, phases[slave - 1]
 
 
+class PolarisedCatalogue(PlatypusCatalogue):
+    """
+    Extract relevant parts of a polarised PLATYPUS NeXus file
+    for reflectometry reduction
+    """
+
+    def __init__(self, h5d):
+        """
+        Extract relevant parts of a polarised PLATYPUS NeXus file
+        for reflectometry reduction
+        Access information via dict access, e.g. cat['pol_flip_freq'].
+
+        Parameters
+        ----------
+        h5d - HDF5 file handle
+        """
+        super(PolarisedCatalogue, self).__init__(h5d)
+        # Is there a magnet?
+        self.ismagnet = False
+        # Is there a cryocooler?
+        self.iscryo = False
+        # Is there a power supply?
+        self.isPS = False
```

Comment:
what's the power supply for, is that a power supply for sample environment?

Reply:
Yes, a power supply to apply voltages inside the cryostat